packages/bot-web-ui/src/pages/tutorials/guide-content/__tests__/guide-content.spec.tsx

```diff
@@ -127,36 +127,30 @@ describe('<GuideContent />', () => {
     });
 
     it('should trigger the keyDown event upon clicking "Enter"', async () => {
-        const mockEventListener = jest.fn();
+        mock_store.ui.is_desktop = true;
         render(<GuideContent {...mocked_props} />, {
             wrapper,
         });
 
-        document.addEventListener('keydown', mockEventListener);
         const button_cancel = screen.getAllByTestId('tutorials-wrap--tour')[1];
-        // use fireEvent since userEvent doesn't handle the case: userEvent.type(button_cancel, '{Enter}');
-        fireEvent.keyDown(button_cancel, { key: 'Enter', code: 'Enter', keyCode: 13 });
+        userEvent.type(button_cancel, '{enter}');
 
         await waitFor(() => {
-            expect(mockEventListener).toHaveBeenCalledWith(expect.objectContaining({ key: 'Enter', code: 'Enter' }));
+            expect(mock_DBot_store.dashboard.is_tour_dialog_visible).toBeTruthy();
         });
     });
 
-    it('should not trigger the keyDown event upon clicking "Backspace"', async () => {
-        const mockEventListener = jest.fn();
+    it('should not trigger the keyDown event upon clicking "Esc"', async () => {
```

Comment:
@rupato-deriv I added it because one ‘else’ condition didn't pass and the percentage coverage decreased after removing